features/src/main/feature/camel-features.xml

```diff
@@ -1118,23 +1118,26 @@
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient4-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/${httpasyncclient-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.codehaus.jettison/jettison/${jettison-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:joda-time/joda-time/${jodatime2-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:com.google.code.gson/gson/${gson-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/${google-http-client-opencensus-api-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/${google-http-client-opencensus-api-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.google-http-client/${google-http-client-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.guava/${jira-guava-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.atlassian-jira-client/${atlassian-jira-client-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-jira/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-jira' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='[33,34)'>guava</feature>
+        <feature version='4.5.14'>http-client</feature>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.event/atlassian-event/4.1.3</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.httpclient/atlassian-httpclient-library/2.1.5</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.httpclient/atlassian-httpclient-api/2.1.5</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.jira/jira-rest-java-client-core/${jira-rest-client-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.jira/jira-rest-java-client-api/${jira-rest-client-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.sal/sal-api/5.1.4$overwrite=merge&amp;Import-Package=*;resolution:=optional&amp;Export-Package=com.atlassian.sal.api*;version=5.1.4</bundle>
+        <bundle dependency='true'>mvn:io.atlassian.util.concurrent/atlassian-util-concurrent/4.0.1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client/1.43.3$overwrite=merge&amp;Import-Package=io.opencensus*,com.google.api.client*,org.apache.http*,javax*,com.google.common*;version="[33,34)"&amp;Export-Package=com.google.api.client.auth*;version=1.43.3</bundle>
```

Comment:
enforced the new guava

Reply:
Why do you want to enforce the version of guava? Please note that it is not necessary in OSGI, you can have several versions of guava as a dependencies of the same feature

Reply:
Here as you have an OSGI bundle that explicitly expects a specific version of Guava, I would rather prefer not to enforce the version of Guava

Reply:
removed the Guava version enforcement, thanks!

Reply:
it is not exactly what I meant sorry for the misunderstanding, the idea is to avoid using the wrap protocol for OSGI bundle unless you have to, in order to fix a bug in the manifest or a CVE. Is it the case here?

Reply:
When you use the wrap protocol, you alter the manifest, which should be done only when really needed

Reply:
This bundle requires
`com.google.common.b
 ase;version="[30.1,31)",com.google.common.collect;version="[30.1,31)"
 ,com.google.common.io;version="[30.1,31)",com.google.common.util.conc
 urrent;version="[30.1,31)"`

Latest guava suitable is https://mvnrepository.com/artifact/com.google.guava/guava/30.1.1-jre (it does have 2 CVEs)
As we discussed earlier, the preferred version of guava is latest, this is the reason wrap is used and guava version was enforced
we can as well add an older guava as a feature. However, the dependency tree does not suggest that either

Reply:
com.google.oauth-client requires guava 31.1
`[com.google.oauth-client/1.35.0] osgi.wiring.package; filter:="(&(osgi.wiring.package=com.google.common.base)(version>=31.1.0)(!(version>=32.0.0)))"]]]`
google-http-client requires guava 30.1
`[com.google.http-client.google-http-client/1.43.3] osgi.wiring.package; filter:="(&(osgi.wiring.package=com.google.common.base)(version>=30.1.0)(!(version>=31.0.0)))"]]]`
camel-jira requires the latest guava - enforced by editing the component's pom because the dependecy tree in camel project suggest that version of guava
` [camel-jira/4.5.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=com.google.common.base)(version>=33.0.0)(!(version>=34.0.0)))"]]`

So, without wrapping we will end up with 3 guava versions in a single feature, without enforcing the latest guava we end up with 2 old ones.